ui/src/app/cohort-search/redux/reducer.ts

```diff
@@ -385,14 +395,33 @@ export const rootReducer: Reducer<CohortSearchState> =
           .setIn(['wizard', 'item', 'attributes', 'node'], Map())
           .deleteIn(['wizard', 'calculate', 'count']);
 
+      case HIDE_ITEM: {
+        const activeItems = state.getIn(['entities', 'groups', action.groupId, 'items'])
+          .filter(itemId => {
+            const item = getItem(itemId)(state);
+            return item.get('id') !== action.itemId && item.get('status') === 'active';
+          });
+        if (!activeItems.size) {
+          state = state.setIn(['entities', 'groups', action.groupId, 'count'], null);
+        }
+        return state.setIn(['entities', 'items', action.itemId, 'status'], action.status);
+      }
+
+      case HIDE_GROUP:
+        return state.setIn(['entities', 'groups', action.groupId, 'status'], action.status);
```

Comment:
When hiding multiple AND'ed search groups, seems that the cancel isn't called between each hide group call. Total count is calculating wrong.

Reply:
Should be fixed